src/components/schema/types.ts

```diff
@@ -1,4 +1,12 @@
 import { autoCompleteDefaults_Virtual_Multiple } from 'components/shared/AutoComplete/DefaultProps';
+import { InferSchemaResponseProperty } from 'types';
 
 export type OnChange =
     (typeof autoCompleteDefaults_Virtual_Multiple)['onChange'];
+
+export type FieldFilter = 'all' | InferSchemaResponseProperty['exists'];
+
+export interface AutoCompleteOption {
+    id: FieldFilter;
```

Comment:
Curious if `key` would be a more fitting name for this property.

Reply:
I treat `key` as reserved in React